jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/AbstractRuntimeManager.java

```diff
@@ -15,13 +15,15 @@
  */
 package org.jbpm.runtime.manager.impl;
 
+import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.locks.ReentrantLock;
 
+import com.sun.tools.javac.util.ServiceLoader;
```

Comment:
Use java util instead:
```suggestion
import java.util.ServiceLoader;
```

Reply:
removed